x-pack/metricbeat/module/panos/tunnels/_meta/fields.yml

```diff
@@ -0,0 +1,59 @@
+- name: tunnels
+  type: group
+  release: beta
+  description: ipsec tunnels
+  fields:
+    - name: id
+      type: integer
+      description: ID field
+    - name: name
+      type: keyword
+      description: Name field
+    - name: gw
+      type: keyword
+      description: Gateway field
+    - name: TSi_ip
+      type: ip
+      description: TSi IP field
+    - name: TSi_prefix
+      type: integer
+      description: TSi Prefix field
+    - name: TSi_proto
+      type: integer
+      description: TSi Protocol field
+    - name: TSi_port
+      type: integer
+      description: TSi Port field
+    - name: TSr_ip
+      type: ip
+      description: TSr IP field
+    - name: TSr_prefix
+      type: integer
+      description: TSr Prefix field
+    - name: TSr_proto
+      type: integer
+      description: TSr Protocol field
+    - name: TSr_port
+      type: integer
+      description: TSr Port field
+    - name: proto
+      type: integer
+      description: Protocol field
+    - name: mode
+      type: keyword
+      description: Mode field
+    - name: dh
+      type: keyword
+      description: DH field
+    - name: enc
+      type: keyword
+      description: Encryption field
+    - name: hash
+      type: keyword
+      description: Hash field
+    - name: life
+      type: integer
+      description: Life field
+    - name: kb
+      type: integer
+      description: KB field
```

Comment:
```suggestion
- name: tunnels
  type: group
  release: beta
  description: IPsec tunnels information
  fields:
    - name: id
      type: integer
      description: Unique identifier for the tunnel
    - name: name
      type: keyword
      description: Name of the tunnel
    - name: gw
      type: keyword
      description: Gateway IP address
    - name: TSi_ip
      type: ip
      description: Traffic Selector initiator IP address
    - name: TSi_prefix
      type: integer
      description: Traffic Selector initiator prefix length
    - name: TSi_proto
      type: integer
      description: Traffic Selector initiator protocol number
    - name: TSi_port
      type: integer
      description: Traffic Selector initiator port number
    - name: TSr_ip
      type: ip
      description: Traffic Selector responder IP address
    - name: TSr_prefix
      type: integer
      description: Traffic Selector responder prefix length
    - name: TSr_proto
      type: integer
      description: Traffic Selector responder protocol number
    - name: TSr_port
      type: integer
      description: Traffic Selector responder port number
    - name: proto
      type: integer
      description: IPsec protocol number
    - name: mode
      type: keyword
      description: IPsec mode
    - name: dh
      type: keyword
      description: Diffie-Hellman group
    - name: enc
      type: keyword
      description: Encryption algorithm
    - name: hash
      type: keyword
      description: Hash algorithm
    - name: life
      type: integer
      description: Life field
    - name: data_volume_kb
      type: integer
      description: Data volume transferred in kilobytes
```
few questions:

- encryption ones needs to be grouped?
- what is a life?


Please check all fields.yml if some fields can be grouped?

Reply:
Also, please check all the fields.yml and check if:

- Field names are correct
- Types are correct
- Descriptions are correct. Here's a good example for fields.yml: https://github.com/elastic/beats/blob/main/metricbeat/module/vsphere/datastore/_meta/fields.yml